synapse/lib/cell.py

```diff
@@ -60,6 +60,7 @@ class CellApi(s_base.Base):
         self.user = user
         sess = self.link.get('sess')  # type: s_daemon.Sess
         sess.user = user
+        self.link.onfini(self.fini)
```

Comment:
This is already done in daemon.py at https://github.com/vertexproject/synapse/blob/v0.1.47/synapse/daemon.py#L415

```
            if isinstance(item, s_telepath.Aware):
                item = await s_coro.ornot(item.getTeleApi, link, mesg, path)
                if isinstance(item, s_base.Base):
                    link.onfini(item.fini)
```